### PR TITLE
Enable justify option in TinyMCE

### PR DESCRIPTION
**Pull request by maximebiloe**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Based on https://github.com/PrestaShop/PrestaShop/pull/4476/ It enables the `justify` option in TinyMCE |
| Type? | improvement |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| How to test? | You can test on a product in BO |

### CODE REVIEW

File: js/admin/tinymce.inc.js [hunk 1]
```diff
@@ -34,7 +35,7 @@ function tinySetup(config)
 		selector: ".rte" ,
 		plugins : "colorpicker link image filemanager table media placeholder",
 		browser_spellcheck : true,
-		toolbar1 : "colorpicker,bold,italic,underline,strikethrough,blockquote,link,alignleft,aligncenter,alignright,alignfull,bullist,numlist,image",
+		toolbar1 : "colorpicker,bold,italic,underline,strikethrough,blockquote,link,alignleft,aligncenter,alignright,alignjustify,bullist,numlist,image",
```
Comment (mickaelandrieu) [inline js/admin/tinymce.inc.js]: What is the difference between ``alignfull`` and ``alignjustify`` ?

Comment (fojtcz) [inline js/admin/tinymce.inc.js]: i don't see `alignfull` in documentation 

https://www.tinymce.com/docs/advanced/editor-control-identifiers/#toolbarcontrols

Comment (maximebiloe) [inline js/admin/tinymce.inc.js]: `alignfull` never existed ^^

**Comment by mickaelandrieu** [comment]
Ok to me :+1: thanks for the docs link @fojtcz

**Pull request merged**